src/rules/no-entire-object-hook-deps.ts

```
@@ -0,0 +1,196 @@
+        node.expressions.forEach(expr => visit(expr));
+        break;
+    }
+  }
+
+  visit(hookBody);
+
+  // Filter out intermediate paths
+  const paths = Array.from(usages);
+  const filteredPaths = paths.filter(path =>
+    !paths.some(otherPath =>
+      otherPath !== path && otherPath.startsWith(path + '.')
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)